STYLE-GUIDE.md

```diff
@@ -210,6 +210,12 @@ uint256 \_amount
 | --- | --- |
 |  For more information on General Message Passing, check out [Axelar's YouTube channel](https://www.youtube.com/channel/UCf8GFg58fdp1iZwLAOV1Tgg).  |  For more information on General Message Passing, check out Axelar's YouTube channel [here](https://www.youtube.com/channel/UCf8GFg58fdp1iZwLAOV1Tgg).  |
 
+* Use the full path instead of a relative one when linking in docs. This makes links less likely to break if a file is moved into another folder.
+
+|  **Recommended**  |  **Not recommended** |
+| --- | --- |
+|  `Create a [custom token](https://docs.axelar.dev/dev/glossary#custom-token).`  |  `Create a [custom token](../../reference/glossary/#custom-token).`  |
```

Comment:
Can we choose a root-relative path, rather than a full path? aka `/dev/glossary` Otherwise our previews won't work and local dev will be a pain.